tests/org.eclipse.n4js.hlc.tests/src/org/eclipse/n4js/hlc/tests/InstallFromManifestCompileRunN4jscExternalImportsTest.java

```diff
@@ -61,7 +61,7 @@ public void testCompileAndRunWithExternalDependencies() {
 		assertEquals(yarnInstallResult.toString(), 0, yarnInstallResult.getExitCode());
 
 		CliCompileResult cliResult = n4jsc(COMPILE(workspace));
-		assertEquals(cliResult.toString(), 3, cliResult.getTranspiledFilesCount());
+		assertEquals(cliResult.toString(), 7, cliResult.getTranspiledFilesCount());
```

Comment:
Why did the number of compiled files go up?

Reply:
It went up because now the compiler counts all transpiled files, whereas before this number was counted by scanning specific src-gen folders on disk. Since the project test setup puts dependencies into the yarn-packages folders, it also compiles dependencies and adds those transpiled files to this number.